### ISSUE TITLE
Multiple configurations for Active vs Inactive screens (feature request)

### ISSUE THREAD

**Issue by lakshmivyas** [jaor/xmobar#278]
It would be nice to have two configurations that get switched when xmobar's screen gains or loses focus.

**Comment by lakshmivyas** [comment]
To be specific, I'd like to change the background color, border color and possibly the template based on active screen vs this screen.

**Comment by jaor** [comment]
yes, it'd be nice... and one could also then have a signal trigger the configuration switch.  marking as a feature request!
